sdk/invoke.js

```diff
@@ -21,10 +21,19 @@ const invoke = async function invoke(funcName, apiKey, funcData) {
     resolveWithFullResponse: true,
   };
   logger.debug('Invoking function', options);
-  const res = await rp.post(options);
-  const { statusCode, headers, body } = res;
-  logger.debug('Invoke response', { statusCode, headers, body });
-  return res;
+  try {
+    const res = await rp.post(options);
+    const { statusCode, headers, body } = res;
+    logger.debug('Invoke response', { statusCode, headers, body });
+    return res;
+  } catch (err) {
+    const parsedError = JSON.parse(err.error);
```

Comment:
this can throw too, and does not account all possible errors.
the user function can return errors on its own which do not conforms to this pattern.
the code must validates the schema completely before assuming it is a binaris internal error.
(and print user originated errors as well, of course)